redaxo/src/addons/cronjob/lib/form.php

```diff
@@ -68,6 +74,17 @@ public function setValue($value)
         $this->value = $value;
     }
 
+    /**
+     * @return array
```

Comment:
Hier wird ein array returned, in parent: getValue() aber definiert dass ein string returned werden muss/soll... hier müssen wir obacht geben